src/pages/projectSettings/tabs/AccessTab/getFormSchema.ts

```diff
@@ -68,11 +68,16 @@ export const getFormSchema = (
 });
 
 const getAdminsDescription = (projectType: ProjectType): string => {
+  let description = "";
   if (projectType === ProjectType.Repo) {
-    return "Admins for this repo will be able to edit repo settings and any attached branches’ settings.";
+    description =
+      "Admins for this repo will be able to edit repo settings and any attached branches’ settings.";
+  } else if (projectType === ProjectType.AttachedProject) {
+    description =
+      "Admins for this branch will be able to edit branch settings and view repo settings.";
+  } else {
+    description =
+      "Admins for this branch will be able to edit branch settings.";
   }
```

Comment:
You can clean this up a little bit by using a map or a switch with the descriptions.
```ts
const getAdminsDescription = (projectType: ProjectType): string => {
  const descriptions = {
    [ProjectType.Repo]: "Admins for this repo will be able to edit repo settings and any attached branches’ settings.",
    [ProjectType.AttachedProject]: "Admins for this branch will be able to edit branch settings and view repo settings.",
    default: "Admins for this branch will be able to edit branch settings."
  };
  const description = descriptions[projectType] || descriptions.default;
  return `${description} All admins will have access to create new projects on Evergreen.`;
};
```
```ts
const getAdminsDescription = (projectType: ProjectType): string => {
  let description = "";
  switch (projectType) {
    case ProjectType.Repo:
      description = "Admins for this repo will be able to edit repo settings and any attached branches’ settings.";
      break;
    case ProjectType.AttachedProject:
      description = "Admins for this branch will be able to edit branch settings and view repo settings.";
      break;
    default:
      description = "Admins for this branch will be able to edit branch settings.";
      break;
  }
  return `${description} All admins will have access to create new projects on Evergreen.`;
};
```